members/README.md

```diff
@@ -91,3 +91,67 @@ Returns all inactive/idle members in the system.
 - **Error Response:**
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'Something went wrong. Please contact admin' }`
+
+
+
+------
+# Getting Started with Recruiter Introductory Forms
+
+## **Recruiter Object**
+
+```
+{
+    'company': string,
+    'fname': string,
+    'lname': string,
+    'designation': string,
+    'reason': string,
+    'email': string,
+    'currency': string
+    'package': number
```

Comment:
`designation`, who's designation? Like the recruiter's or the designation for which they're hiring?

Also, line number 110 is missing trailing comma

Reply:
It's the recruiter's designation. As this key belongs to the recruiter object, I think it's implied that it is asking for the recruiter's designation😅😅.